go/tools/git/git.go

```diff
@@ -33,4 +33,13 @@ func GetCommitHash(repoDir string) (hash string, err error) {
 
 	hash = ref.Hash().String()
 	return hash, nil
+}
+
+// ShortenSHA will return the first 7 characters of a SHA.
+// If the given SHA is too short, it will be returned untouched.
+func ShortenSHA(sha string) string {
+	if len(sha) > 7 {
```

Comment:
is 7 enough for us?

Reply:
`ShortenSHA` will return the commit SHA like `git` does, instead of `69eaf6578535a1b053ae9f84bc2e2d7530738c8f` they become `69eaf65`, 7 is the length of the latter string

Reply:
In the web UI we will use both long and short strings. The shorter string will be used to render information and avoid having a long text. The longer string will be used by hyperlinks and will redirect users to vitessio/vitess.